libfwupdplugin/fu-udev-device.c

```diff
@@ -1269,15 +1269,8 @@ fu_udev_device_set_physical_id(FuUdevDevice *self, const gchar *subsystems, GErr
 	}
 
 	if (g_strcmp0(subsystem, "pci") == 0) {
-		tmp = g_udev_device_get_property(udev_device, "PCI_SLOT_NAME");
-		if (tmp == NULL) {
-			g_set_error_literal(error,
-					    G_IO_ERROR,
-					    G_IO_ERROR_NOT_FOUND,
-					    "failed to find PCI_SLOT_NAME");
-			return FALSE;
-		}
-		physical_id = g_strdup_printf("PCI_SLOT_NAME=%s", tmp);
+		tmp = g_udev_device_get_sysfs_path(udev_device);
+		physical_id = g_strdup(tmp);
```

Comment:
this isn't a physical ID tho -- this is a backend_id -- the physical ID is supposed to be the same for each physical device, even if it has two different sysfs-paths being added by more than and backend. I think this change has the potential to cause regressions IMHO. That said, I think I can see what you're doing -- and maybe we can use the backend ID instead.

Reply:
@superm1 I've added a fixup -- can you tell me if that works for you?

Reply:
yup it works

Reply:
Done!